boardgames/web/templates/page/index.html.eex

```diff
@@ -0,0 +1,60 @@
+<div class="panel-group">
```

Comment:
You need to put columns inside rows:

~~~ html
<div class="row">
  <div class="col-sm-6">
     ...

~~~